selectors/benefits.js

```diff
@@ -183,40 +181,35 @@ export const getFilteredBenefits = createSelector(
 );
 
 export const getFilteredNextSteps = createSelector(
-  [
-    getNextSteps,
-    getEligibilityPaths,
-    getProfileFilters,
-    getMultipleChoiceOptions,
-    getQuestions
-  ],
-  (nextSteps, eligibilityPaths, profileFilters, multipleChoiceOptions) => {
-    // find next steps that match
-    let eligibleNextStepIds = [];
-    eligibilityPaths.forEach(ep => {
-      if (eligibilityMatch(ep, profileFilters, multipleChoiceOptions)) {
-        eligibleNextStepIds = eligibleNextStepIds.concat(ep.nextSteps);
-      }
-    });
+  [getNextSteps, getProfileFilters, getMultipleChoiceOptions, getQuestions],
+  (nextSteps, profileFilters, multipleChoiceOptions) => {
     let hasSelections = false;
 
     Object.keys(profileFilters).forEach(criteria => {
       if (profileFilters[criteria] !== "") hasSelections = true;
     });
 
-    if (hasSelections)
+    if (hasSelections) {
+      // only check eligible nextSteps if we know we have selections
+      let eligibleNextStepIds = [];
+      nextSteps.forEach(ns => {
+        if (
+          benefitEligibilityMatch(ns, profileFilters, multipleChoiceOptions)
```

Comment:
maybe we should rename this function to be `eligibilityMatch` if it's going to be used for next steps too.

Reply:
I think that this could be done as part of an upcoming PR to remove eligibilityPaths from the code.